sdk/communication/azure-communication-networktraversal/src/main/java/com/azure/communication/networktraversal/CommunicationRelayAsyncClient.java

```diff
@@ -35,6 +36,16 @@
         client = communicationNetworkingClient.getCommunicationNetworkTraversals();
     }
 
+    /**
+     * Creates a new CommunicationRelayConfiguration.
```

Comment:
The method name is `getRelayConfiguration` but javadoc describes this as creating a config. Update the javadoc.